docs/userguide/datetime/calendar/examples.md

```diff
@@ -26,35 +26,36 @@ gc->set(Calendar::SECOND, 0);
 gc->set(Calendar::MILLISECOND, 0);
 // Iterate through the days and print it out.
 for (int32_t i = 0; i < 30; i++) {
-// print out the date.
-// You should use the DateFormat to properly format it
-printf("year: %d, month: %d (%d in the implementation), day: %d\\n",
-gc->get(Calendar::YEAR, status),
-gc->get(Calendar::MONTH, status) + 1,
-gc->get(Calendar::MONTH, status),
-gc->get(Calendar::DATE, status));
-if (U_FAILURE(status))
-{
-puts("Calendar::get failed");
-return;
-}
-// Add a day to the date
-gc->add(Calendar::DATE, 1, status);
-if (U_FAILURE(status)) {
-puts("Calendar::add failed");
-return;
-}
+  // print out the date.  
+  // You should use the DateFormat to properly format it
+  printf("year: %d, month: %d (%d in the implementation), day: %d\\n",
+  gc->get(Calendar::YEAR, status),
+  gc->get(Calendar::MONTH, status) + 1,
+  gc->get(Calendar::MONTH, status),
+  gc->get(Calendar::DATE, status));
+  if (U_FAILURE(status))
+  {
+      puts("Calendar::get failed");
+      return;
+  }
+  // Add a day to the date
+  gc->add(Calendar::DATE, 1, status);
+  if (U_FAILURE(status)) {
+    puts("Calendar::add failed");
+    return;
+  }
 }
 delete gc;
+````
 
 **C**
-
+````C
 UErrorCode status = U_ZERO_ERROR;
 int32_t i;
 UCalendar \*cal = ucal_open(NULL, -1, NULL, UCAL_GREGORIAN, &status);
```

Comment:
remove backslash -- please check these files for `\*` and fix all as appropriate

Reply:
Done